radix-engine-tests/tests/metering.rs

```diff
@@ -695,3 +695,18 @@ struct TestNonFungibleData {
 impl NonFungibleData for TestNonFungibleData {
     const MUTABLE_FIELDS: &'static [&'static str] = &["metadata"];
 }
+
+#[test]
+/// The large_package blueprint combines toogether two other packages just to provide meaningful content for
+/// a large package of size as close as possible to current limit: 1048576 bytes minus the size of
+/// SCRYPTO_SBOR_V1_PAYLOAD_PREFIX and the actor size.
+/// If this test fails with an error TransactionLimitsError::MaxInvokePayloadSizeExceeded,
+/// go to `blueprints/large_package/Cargo.toml` file and change `package2` reference package name and path
+/// some other blueprint, but verify transaction payload size if it is close enough to the limit (it can be
+/// done by checking TxPayloadCost from cost breakdown table divided by payload byte cost, which can be taken
+/// from fee_table.rs tx_payload_cost() function). List of blueprints and its size can be displayed using command
+/// `ls -lSk ./radix-engine-tests/tests/blueprints/target/wasm32-unknown-unknown/release/*.wasm`
+fn publish_package_1mib() {
+    let mut test_runner = TestRunnerBuilder::new().build();
+    test_runner.compile_and_publish("./tests/blueprints/large_package");
```

Comment:
It would be nicer if we can make the wasm a static asset, to avoid flaky results caused by host environment.